core/generated-sources/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/columnRange.kt

```diff
@@ -0,0 +1,66 @@
+package org.jetbrains.kotlinx.dataframe.api
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.matchers.shouldBe
+import org.jetbrains.kotlinx.dataframe.samples.api.age
+import org.jetbrains.kotlinx.dataframe.samples.api.city
+import org.jetbrains.kotlinx.dataframe.samples.api.firstName
+import org.jetbrains.kotlinx.dataframe.samples.api.lastName
+import org.jetbrains.kotlinx.dataframe.samples.api.name
+import org.jetbrains.kotlinx.dataframe.samples.api.weight
+import org.junit.Test
+
+class ColumnRangeTest : ColumnsSelectionDslTests() {
+
+    @Test
+    fun `top level columns`() {
+        listOf(
+            df.select { age and city and weight },
+
+            df.select { "age".."weight" },
+            df.select { "age"..Person::weight },
+            df.select { "age"..weight },
+            df.select { "age"..pathOf("weight") },
+
+            df.select { Person::age.."weight" },
+            df.select { Person::age..Person::weight },
+            df.select { Person::age..weight },
+            df.select { Person::age..pathOf("weight") },
+
+            df.select { age.."weight" },
+            df.select { age..Person::weight },
+            df.select { age..weight },
+            df.select { age..pathOf("weight") },
+
+            df.select { pathOf("age").."weight" },
+            df.select { pathOf("age")..Person::weight },
```

Comment:
It's not forbidden. The overloads just didn't exist before. I think it makes a lot of sense to be able to mix APIs, especially with how DF currently works without compiler plugin: Say you have a dataframe where you just added a new column:
```kotlin
df.add("newCol") { 0 }
  .select { existingCol.."newCol" }
```
there's no accessor for newCol yet, so you need to reference it in another way. Sure, a user can add `newCol` by a column accessor, however, if they choose the String API is good enough for their new column, they should not be forced to use a different API suddenly or worse, have to downgrade to `"existingCol".."newCol"` just because we didn't provide the overloads.